src/control/controlbehavior.cpp

```diff
@@ -161,19 +171,19 @@ double ControlAudioTaperPotBehavior::valueToParameter(double dValue) {
     } else if (dValue < 1.0) {
         // db + linear overlay to reach
         // m_minDB = 0
-        // 0 dB = m_neutralParameter
+        // 0 dB = m_dNeutralParameter
         double overlay = m_offset * (1 - dValue);
         if (m_minDB) {
-            dParam = (ratio2db(dValue + overlay) - m_minDB) / m_minDB * m_neutralParameter * -1;
+            dParam = (ratio2db(dValue + overlay) - m_minDB) / m_minDB * m_dNeutralParameter * -1;
```

Comment:
Oh, sorry I did not notice before. We have a naming issue here. 
ControlAudioTaperPotBehavior::m_neutralParameter != ControlPotmeterBehavior::m_dNeutralParameter

ControlAudioTaperPotBehavior::m_neutralParameter: 
The parameter, that leads to a neutral gain of 1 = 0.dB 

ControlPotmeterBehavior::m_dNeutralParameter: 
The parameter, that hides the ring entirely. 

I think we should rename both, to something like this: 
ControlAudioTaperPotBehavior::m_zeroDbParameter.
ControlPotmeterBehavior::m_scaleStartParameter.

Do you use a refactoring tool? This will perform all renaming automatically.